extensions/wikia/ArticleVideo/ArticleVideoController.class.php

```diff
@@ -12,9 +12,15 @@ public function featured() {
 
 			$this->setVal( 'videoDetails', $featuredVideoData );
 			if ( ArticleVideoContext::isJWPlayer( $featuredVideoData ) ) {
-                $this->setVal( 'jwplayerInstance', file_get_contents($this->getApp()->wg->extensionsPath . '/wikia/ArticleVideo/scripts/featured-video.jwplayer.instance.js') );
-				$this->setVal( 'featuredVideoAutoplay', file_get_contents($this->getApp()->wg->extensionsPath . '/wikia/ArticleVideo/scripts/featured-video.autoplay.js') );
-				$this->setVal( 'jwplayerScript', file_get_contents($this->getApp()->wg->extensionsPath . '/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js') );
+				$jwPlayerScript = RequestContext::getMain()->getOutput()->getResourceLoader()->getModule( 'ext.ArticleVideo.jw' )->getScript(
+					new \ResourceLoaderContext( new \ResourceLoader(), \RequestContext::getMain()->getRequest())
+				);
+
+				$this->setVal(
+					'jwPlayerScript',
+					\AssetsManagerBaseBuilder::minifyJS($jwPlayerScript)
```

Comment:
That is a weird combo :)
Can we use one RL or AM?

Reply:
OK, I used a generic minifier